### PR TITLE
extracted cdc

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -8,15 +8,19 @@ eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-rel
 springBootVersion=1.4.7.RELEASE
 springBootCdcVersion=2.1.1.RELEASE
 
-eventuateClientVersion=0.21.1.RELEASE
-eventuateLocalVersion=0.30.2.RELEASE
+eventuateLocalVersion=0.15.0-SNAPSHOT
```
Comment (cer) [inline gradle.properties]: Does it need to use Eventuate Local? Or have things moved to Common

File: gradle.properties [hunk 2]
```diff
@@ -8,15 +8,19 @@ eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-rel
...
 eventuateUtilVersion=0.1.0.RELEASE
 
 springCloudContractDependenciesVersion=2.0.0.RELEASE
 springDependencyManagementPluginVersion=1.0.3.RELEASE
 
 springCloudSleuthVersion=2.0.2.RELEASE
 dockerComposePluginVersion=0.6.6
-
-version=0.21.0-SNAPSHOT
+eventuateCdcVersion=0.1.0-SNAPSHOT
```
Comment (cer) [inline gradle.properties]: Eventuate CDC should only be used in tests (src/tests) or as an image version

File: gradle.properties [hunk 3]
```diff
@@ -8,15 +8,19 @@ eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-rel
...
+eventuateCommonVersion=0.4.0.RELEASE
+eventuateMessagingKafkaVersion=0.1.0-SNAPSHOT
+eventuateMessagingActiveMQVersion=0.1.0-SNAPSHOT
+eventuateMessagingRabbitMQVersion=0.1.0-SNAPSHOT
+eventuateMessagingRedisVersion=0.1.0-SNAPSHOT
+version=0.20.0.REDIS-SNAPSHOT
```
Comment (cer) [inline gradle.properties]: This is no longer REDIS-SNAPSHOT

File: eventuate-tram-consumer-jdbc/build.gradle [hunk 4]
```diff
@@ -4,7 +4,7 @@ dependencies {
     compile project(":eventuate-tram-consumer-common")
     compile project(":eventuate-tram-jdbc")
 
-    compile "io.eventuate.local.java:eventuate-local-java-jdbc:$eventuateLocalVersion"
+    compile "io.eventuate.cdc:eventuate-local-java-jdbc:$eventuateCdcVersion"
```
Comment (cer) [inline eventuate-tram-consumer-jdbc/build.gradle]: Eventuate Tram (src/main)  should not depend on Eventuate CDC - SqlDialect should be moved into Common?

File: gradle.properties [hunk 5]
```diff
@@ -8,15 +8,19 @@ eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-rel
...
 springCloudContractDependenciesVersion=2.0.0.RELEASE
 springDependencyManagementPluginVersion=1.0.3.RELEASE
 
 springCloudSleuthVersion=2.0.2.RELEASE
 dockerComposePluginVersion=0.6.6
-
-version=0.21.0-SNAPSHOT
+eventuateCdcVersion=0.1.0-SNAPSHOT
+eventuateCommonVersion=0.4.0.RELEASE
+eventuateMessagingKafkaVersion=0.1.0-SNAPSHOT
```
Comment (cer) [inline gradle.properties]: Please update with the released versions

File: gradle.properties (hunk 1 above)
Comment (dartartem) [reply gradle.properties]: I will check.

File: gradle.properties (hunk 2 above)
Comment (dartartem) [reply gradle.properties]: I will fix

File: gradle.properties (hunk 3 above)
Comment (dartartem) [reply gradle.properties]: I will fix

File: eventuate-tram-consumer-jdbc/build.gradle (hunk 4 above)
Comment (dartartem) [reply eventuate-tram-consumer-jdbc/build.gradle]: I will fix

File: gradle.properties (hunk 5 above)
Comment (dartartem) [reply gradle.properties]: ok, will do

File: docker-compose-mssql-polling.yml [hunk 6]
```diff
@@ -1,13 +1,13 @@
 zookeeper:
-  image: eventuateio/eventuateio-local-zookeeper:0.30.1.RELEASE
+  image: test-eventuate-zookeeper
```
Comment (cer) [inline docker-compose-mssql-polling.yml]: More test images

**Pull request merged**